zkevm-circuits/src/evm_circuit/execution.rs

```diff
@@ -77,7 +78,7 @@ mod sstore;
 mod stop;
 mod swap;
 
-use self::sha3::Sha3Gadget;
+use self::{returndatacopy::ReturnDataCopyGadget, sha3::Sha3Gadget};
```

Comment:
Import looks a bit strange compared to the other gadget imports because of the `self::`. Would just do `use returndatacopy::ReturnDataCopyGadget;` on its own line like all the other gadgets.

Reply:
fixed